src/lighteval/tasks/lighteval_task.py

```diff
@@ -189,16 +188,6 @@ def __init__(  # noqa: C901
 
         current_categories = [metric.category for metric in self.metrics]
         self.has_metric_category = {category: (category in current_categories) for category in MetricCategory}
```

Comment:
Why did you remove this check?

Reply:
because we do not always use open ai for llm as judge tasks now

Reply:
:bulb: